rbd.c

```diff
@@ -115,11 +115,12 @@ static darray(char *) blacklist_caches;
 #ifdef LIBRADOS_SUPPORTS_SERVICES
 
 #ifdef RBD_LOCK_ACQUIRE_SUPPORT
+/* rdev->priv_lock is held_*/
```

Comment:
```suggestion
/* rdev->priv_lock is held */
```
here and everywhere else

Reply:
Will fix it.